### ISSUE TITLE
Not working in Android P Webview browser

### ISSUE THREAD

**Issue by jaweii** [fabricjs/fabric.js#4964]
![1](https://user-images.githubusercontent.com/22613580/39905118-49580dd6-550e-11e8-817e-f628ad56dd2f.png)

**Comment by asturur** [comment]
not working in the emulator of a developer preview of android?

**Comment by jaweii** [comment]
Yes, in my emulator, it does not work when the android version >= 7.0. 
( It is only not work in webview browser, it works fine in chrome )

**Comment by asturur** [comment]
his is a screenshot from 7.1.2 phone, the internal browser that i think
uses the webview. On a real phone works.

Consider that if it does not work is likely the webview having problems,
with maybe webgl? try something where there are no image filters, try a
simple demo.
